frontend/src/components/Item/CollectionTabs.vue

```diff
@@ -19,46 +19,33 @@
   </div>
 </template>
 
-<script lang="ts">
+<script setup lang="ts">
```

Comment:
Github also destroyed this diff too. No logic changes, just conversion to composition setup api